.github/workflows/ci.yml

```diff
@@ -424,10 +172,61 @@ jobs:
         path: ./build/f3d-*
         name: f3d-${{matrix.os}}-${{matrix.vtk_version}}
 
+#----------------------------------------------------------------------------
+# Coverage: Build and test on linux with last VTK with coverage option
+#----------------------------------------------------------------------------
+  coverage:
```

Comment:
can we split the regular CI and coverage in two different files?

Reply:
Not if we want a single workflow